### PR TITLE
docs: fix typos and improve style in README.md

### PR DESCRIPTION
**Pull request by dkocich**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,7 +3,7 @@
 [![Build Status](https://travis-ci.org/Wykks/ngx-mapbox-gl.svg?branch=master)](https://travis-ci.org/Wykks/ngx-mapbox-gl)
 [![npm version](https://img.shields.io/npm/v/ngx-mapbox-gl.svg?style=flat)](https://www.npmjs.com/package/ngx-mapbox-gl)
 
-Angular wrapper for [mapbox-gl-js](https://www.mapbox.com/mapbox-gl-js/api/). Expose a bunch of component meant to be simple to use for Angular.
+Angular wrapper for [mapbox-gl-js](https://www.mapbox.com/mapbox-gl-js/api/). Expose a bunch of components meant to be simple to use for Angular.
```
Comment (dmytro-gokun) [inline README.md:6]: I think it should say "Exposes".

Comment (dmytro-gokun) [reply README.md:6]: Also "for Angular" should be "with Angular"

File: README.md [hunk 2]
```diff
@@ -59,7 +59,7 @@ For example, with angular-cli add this in angular.json
       ],
 ```
 
-Or in global css (called styles.css for example in angular-cli)
+Or in global CSS (called styles.css for example in angular-cli)
```
Comment (dmytro-gokun) [inline README.md:62]: Or in **the** global CSS **file**

File: README.md [hunk 3]
```diff
@@ -92,7 +92,7 @@ export class AppModule {}
 
 How to get a mapbox token: https://www.mapbox.com/help/how-access-tokens-work/
 
-Note: mapbox-gl can works without token, if you have your own source, example: https://stackblitz.com/edit/ngx-mapbox-gl-without-token
+Note: mapbox-gl works without a token, if you have your own source, example: https://stackblitz.com/edit/ngx-mapbox-gl-without-token
```
Comment (dmytro-gokun) [inline README.md:95]: works = can work

File: README.md [hunk 4]
```diff
@@ -182,9 +182,9 @@ export class MyLibMapModule {
 }
 ```
 
-We basically create a `forRoot` static function in the library module, that will accept a *configuration* object as parameter. This *configuration* will provide the actual token to the `ngx-mapbox-gl` module via providers by providing the value from the *configuration* to the `MAPBOX_API_KEY` injection token.
+We basically create a `forRoot` static function in the library module, that will accept a *configuration* object as  a parameter. This *configuration* will provide the actual token to the `ngx-mapbox-gl` module via providers by providing the value from the *configuration* to the `MAPBOX_API_KEY` injection token.
 
-Finally in the application that will use your `MyLibMapModule`, you will import the module in this way:
+Finally, in the application that will use your `MyLibMapModule`, you will import the module in this way:
```
Comment (dmytro-gokun) [inline README.md:187]: in this way = this way

**Comment by dkocich** [comment]
I do not want to be too perfectionist here to discuss and update all the grammar because I am also not a native speaker - my main point of this change was wrong code 'mglFullscren' which I did not notice and it crashed my app however when I did this change, I did not want to submit one letter fix so I quickly tried to revise all of the content... I will revise and update your changes later :)

**Review by dmytro-gokun — APPROVED** [review]
This is perfect!

**Comment by dmytro-gokun** [comment]
@dkocich Thanks a lot for your time and effort!

**Pull request merged**